utils/Room.js

```diff
@@ -0,0 +1,31 @@
+module.exports = class Room{
+    #private = 123
+    constructor(){
+        this.listOfPlayers = []
+        this.nextPlayer = 0;
+    }
+    incrementPlayer(){
```

Comment:
its incrementing the player number not the player
make the function name better